src/parsers/manifestjson.js

```diff
@@ -234,6 +236,27 @@ export default class ManifestJSONParser extends JSONParser {
     }
   }
 
+  async validateIcon(_path, size) {
```

Comment:
given that we are extracting this into a new method I would also remove the `_` from `_path` and rename `size` into `expectedSize`. 

I think that it makes sense to make it a method, @EnTeQuAk how that sounds to you?

(if we opt to extract it into this new `validateIcon` method, we should also add a new test case for it in the same pull request).

Reply:
> I think that it makes sense to make it a method, @EnTeQuAk how that sounds to you?

Agreed, `_path` is a weird argument name for a method that should have a nicer interface.